docs/components.ipynb

```diff
@@ -0,0 +1,386 @@
+   "id": "68e35c62-6072-47b1-8c3e-e610826da008",
+   "metadata": {},
+   "source": [
+    "## Multiple choppers\n",
+    "\n",
+    "It is of course possible to add more than one chopper.\n",
+    "Here we add a second one, further down the beam path,\n",
+    "which splits each groups into two more groups."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ac5f9697-a2a6-45a8-ad99-af6ad1a60be7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "chopper2 = tof.Chopper(\n",
+    "    frequency=20.0 * Hz,\n",
+    "    open=sc.array(\n",
+    "        dims=['cutout'],\n",
+    "        values=[60.0, 100.0, 160.0, 220.0],\n",
+    "        unit='deg',\n",
+    "    ),\n",
+    "    close=sc.array(\n",
+    "        dims=['cutout'],\n",
+    "        values=[90.0, 140.0, 200.0, 260.0],\n",
+    "        unit='deg',\n",
+    "    ),\n",
+    "    phase=60.0 * deg,\n",
+    "    distance=20 * meter,\n",
+    "    name=\"Chopper2\",\n",
+    ")\n",
+    "\n",
+    "model.choppers = [chopper1, chopper2]\n",
+    "model.run()\n",
+    "model.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "62d721c5-3c3e-457c-ab05-60659fccdaaa",
+   "metadata": {},
+   "source": [
+    "The distribution of neutrons that are blocked and pass through the second chopper looks as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab11bd09-73ba-4bf8-9b52-e503d77f3f5c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "chopper2.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "09d6cbef-f2f3-48db-a696-b4ed912817c8",
+   "metadata": {},
+   "source": [
+    "While the detector now sees 4 peaks"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20349f5e-bfd3-4ccb-bde4-119f3e3d423a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "detector.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55c8ca14-e885-43b2-8409-48c1eb964625",
+   "metadata": {},
+   "source": [
+    "To view the blocked rays on the time-distance diagram of the model, use"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a3e040c6-afee-495f-a32f-7fdcad506df1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model.plot(max_rays=100, blocked_rays=5000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b3c4a5fb-010b-4d6a-a391-4fb48aa4b1d2",
+   "metadata": {},
+   "source": [
+    "## Adding a monitor\n",
+    "\n",
+    "Detectors can be placed anywhere in the beam path,\n",
+    "and in the next example we place it between the first and second choppers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1c8bdc32-583a-4c7c-8c6d-34dab8061c6f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "monitor = tof.Detector(distance=15.0 * meter, name='monitor')\n",
+    "\n",
+    "model.detectors = [monitor, detector]\n",
```

Comment:
Does the order matter?

Reply:
No, components get sorted according to `.distance`.